resources/tests/test_resources_queries.py

```diff
@@ -1267,3 +1268,139 @@ def test_berth_is_invoiceable(api_client):
 
     executed = api_client.execute(query % "false")
     assert executed["data"] == {"berths": {"count": 1, "totalCount": 11}}
+
+
+def test_get_harbor_services(api_client):
+    first_pier = PierFactory(
+        mooring=True,
+        water=True,
+        electricity=True,
+        waste_collection=True,
+        gate=True,
+        lighting=False,
+        suitable_boat_types__count=2,
+    )
+    second_pier = PierFactory(
+        harbor=first_pier.harbor,
+        mooring=False,
+        water=False,
+        electricity=False,
+        waste_collection=False,
+        gate=False,
+        lighting=False,
+        suitable_boat_types__count=2,
+    )
+    harbor = Harbor.objects.get(id=first_pier.harbor.id)
+
+    combined_types = list(
+        first_pier.suitable_boat_types.values_list("id", flat=True)
+    ) + list(second_pier.suitable_boat_types.values_list("id", flat=True))
+
+    suitable_boat_types = [
+        {"id": str(type_id)} for type_id in sorted(set(combined_types))
```

Comment:
Does the backend always return the results in sorted order?

Reply:
It does:
```python
type_ids = (
    self.piers.order_by("suitable_boat_types__id")
    .distinct("suitable_boat_types__id")
    .values_list("suitable_boat_types", flat=True)
)
```